┏━━━┳━━━━━━━━━┳━━━━━━━━━━━━━━━┳━━━━━━━━┓
┃[1m   [22m┃[1m Name    [22m┃[1m Type          [22m┃[1m Params [22m┃
┡━━━╇━━━━━━━━━╇━━━━━━━━━━━━━━━╇━━━━━━━━┩
│ 0 │ encoder │ ResNetEncoder │ 11.2 M │
│ 1 │ decoder │ ResNetDecoder │  8.6 M │
│ 2 │ fc      │ Linear        │  131 K │
└───┴─────────┴───────────────┴────────┘
[1mTrainable params[22m: 19.9 M
[1mNon-trainable params[22m: 0
[1mTotal params[22m: 19.9 M
[1mTotal estimated model params size (MB)[22m: 79
GPU available: True, used: False
TPU available: False, using: 0 TPU cores
IPU available: False, using: 0 IPUs
HPU available: False, using: 0 HPUs
/home/mila/s/sonnery.hugo/.conda/envs/env-rl-training/lib/python3.7/site-packages/pytorch_lightning/trainer/configuration_validator.py:386: LightningDeprecationWarning: The `Callback.on_epoch_end` hook was deprecated in v1.6 and will be removed in v1.8. Please use `Callback.on_<train/validation/test>_epoch_end` instead.
  f"The `Callback.{hook}` hook was deprecated in v1.6 and"
Traceback (most recent call last):
  File "train.py", line 335, in <module>
    main(args)
  File "train.py", line 299, in main
    dm.val_dataloader(),
  File "/home/mila/s/sonnery.hugo/.conda/envs/env-rl-training/lib/python3.7/site-packages/pytorch_lightning/trainer/trainer.py", line 769, in fit
    self._fit_impl, model, train_dataloaders, val_dataloaders, datamodule, ckpt_path
  File "/home/mila/s/sonnery.hugo/.conda/envs/env-rl-training/lib/python3.7/site-packages/pytorch_lightning/trainer/trainer.py", line 721, in _call_and_handle_interrupt
    return trainer_fn(*args, **kwargs)
  File "/home/mila/s/sonnery.hugo/.conda/envs/env-rl-training/lib/python3.7/site-packages/pytorch_lightning/trainer/trainer.py", line 809, in _fit_impl
    results = self._run(model, ckpt_path=self.ckpt_path)
  File "/home/mila/s/sonnery.hugo/.conda/envs/env-rl-training/lib/python3.7/site-packages/pytorch_lightning/trainer/trainer.py", line 1222, in _run
    self._log_hyperparams()
  File "/home/mila/s/sonnery.hugo/.conda/envs/env-rl-training/lib/python3.7/site-packages/pytorch_lightning/trainer/trainer.py", line 1290, in _log_hyperparams
    logger.log_hyperparams(hparams_initial)
  File "/home/mila/s/sonnery.hugo/.conda/envs/env-rl-training/lib/python3.7/site-packages/pytorch_lightning/utilities/rank_zero.py", line 32, in wrapped_fn
    return fn(*args, **kwargs)
TypeError: log_hyperparams() missing 1 required positional argument: 'params'
Traceback (most recent call last):
  File "train.py", line 335, in <module>
    main(args)
  File "train.py", line 299, in main
    dm.val_dataloader(),
  File "/home/mila/s/sonnery.hugo/.conda/envs/env-rl-training/lib/python3.7/site-packages/pytorch_lightning/trainer/trainer.py", line 769, in fit
    self._fit_impl, model, train_dataloaders, val_dataloaders, datamodule, ckpt_path
  File "/home/mila/s/sonnery.hugo/.conda/envs/env-rl-training/lib/python3.7/site-packages/pytorch_lightning/trainer/trainer.py", line 721, in _call_and_handle_interrupt
    return trainer_fn(*args, **kwargs)
  File "/home/mila/s/sonnery.hugo/.conda/envs/env-rl-training/lib/python3.7/site-packages/pytorch_lightning/trainer/trainer.py", line 809, in _fit_impl
    results = self._run(model, ckpt_path=self.ckpt_path)
  File "/home/mila/s/sonnery.hugo/.conda/envs/env-rl-training/lib/python3.7/site-packages/pytorch_lightning/trainer/trainer.py", line 1222, in _run
    self._log_hyperparams()
  File "/home/mila/s/sonnery.hugo/.conda/envs/env-rl-training/lib/python3.7/site-packages/pytorch_lightning/trainer/trainer.py", line 1290, in _log_hyperparams
    logger.log_hyperparams(hparams_initial)
  File "/home/mila/s/sonnery.hugo/.conda/envs/env-rl-training/lib/python3.7/site-packages/pytorch_lightning/utilities/rank_zero.py", line 32, in wrapped_fn
    return fn(*args, **kwargs)
